index.html

```diff
@@ -10,14 +10,11 @@
   specStatus: "ED",
   edDraftURI: "https://w3c.github.io/geolocation-api/",
   editors: [{
-    name: "Andrei Popescu",
-    company: "Google Inc.",
-    companyURL: "https://google.com/",
-    w3cid: "36565"
+    name: "Devices and Sensors Working Group"
```

Comment:
Probably want to move Andrei to `formerEditors:`

Reply:
Oh yes... fixed.